simple_network_sim/loaders.py

```diff
@@ -203,6 +203,75 @@ def readInfectionProbability(df: pd.DataFrame) -> Dict[int, float]:
     return probs
 
 
+def readRandomSeed(df: pd.DataFrame) -> int:
```

Comment:
Assuming the trials will be run in parallel at some point, wouldn't we need one seed per trial?

Reply:
I think you're right in that we might end up in this situation yes. But before making that change I wanted to know what other solutions exist (if any) to the multiple-seeds-not-being-independent issue I mentioned last time. If we proceed with that it should be straightforward enough to change it.

Reply:
For example if we assume only the numbers in the same seed are truly independent, it'd be possible to "burn in" some random number so that each trial runs on a different set from the same seed, e.g.
Trial 1 uses the random numbers (0-1000)
Trial 2 uses the random numbers (1001-2000)
etc

(I don't know if this is real/possible, and I probably wouldn't go for stg more complex than this anyway)

Reply:
The reason I thought about this is that the standard api allows you retrieve a single value, if that's what we want. I've looked into the issue a little bit, maybe we should use this: https://numpy.org/doc/1.17/reference/random/bit_generators/generated/numpy.random.SeedSequence.html that needs just a original seed to create a sequence of seeds, which should be good enough. I stumbled upon this: https://bashtage.github.io/randomgen/parallel.html while I was looking into this.

Reply:
That's great, thanks a lot! I was hoping something like this would exist. I will have a look on how to set this up.